glue/core/roi.py

```diff
@@ -74,6 +74,33 @@ def contains(self, x, y):
         """
         raise NotImplementedError()
 
+    def contains3d(self, x, y, z):
+        """Return true/false for each x/y/z pair.
+
+        Parameters
+        ----------
+
+
+        x : ndarray
```

Comment:
Can you get rid of the empty lines on line 82 and 83?

Reply:
Can you replace `ndarray` by:

```
:class:`numpy.ndarray`
```

Some older docstrings may just have `ndarray` but this doesn't create an intersphinx link